### PR TITLE
Abdi.configure redux store

### PR DESCRIPTION
**Pull request by abdi-edoc-de**
image

### CODE REVIEW

File: starter-project-web-react/store/middlware/apiCall.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            url,
+            method,
+            data,
+            headers
+        })
+
+        dispatch(actions.apiCallSuccess(response.data.json)) // ourt general success action will dispatch heer
+        if (onSuccess) dispatch({ type: onSuccess, payload: response.data.json }) // our passed success action will dispatch  heer if there is one
+    } catch (error) {
+        console.log()
```
Comment (YiseBoge) [inline starter-project-web-react/store/middlware/apiCall.js:24]: better not to have console log

**Review by YiseBoge — APPROVED** [review]
I know nothing about Redux but it looks good enough to me in terms of just code

File: starter-project-web-react/store/middlware/apiCall.js (hunk 1 above)
Comment (abdi-edoc-de) [reply starter-project-web-react/store/middlware/apiCall.js]: done

**Review by Mintaa911 — APPROVED** [review]
It looks good to go.

**Pull request merged**